lldb/source/Plugins/ExpressionParser/Swift/SwiftPersistentExpressionState.h

```diff
@@ -38,7 +41,9 @@ namespace lldb_private {
 /// 0-based counter for naming result variables.
 //----------------------------------------------------------------------
 class SwiftPersistentExpressionState : public PersistentExpressionState {
-  typedef std::set<lldb_private::ConstString> HandLoadedModuleSet;
+
+  typedef std::map<ConstString, swift::AttributedImport<swift::ImportedModule>>
+      HandLoadedModuleSet;
```

Comment:
Do we need the sortedness of std::map or could we use an llvm::DenseMap or perhaps even an llvm::StringMap here?